weblate/static/editor/full.js

```diff
@@ -535,6 +561,11 @@
             '<a class="js-copy-save-machinery btn btn-primary">' +
             gettext("Copy and save") +
             "</a>" +
+            "</td>" +
+            "<td>" +
+            '<a class="js-delete-machinery btn btn-danger">' +
+            gettext("Delete entry") +
+            "</a>" +
```

Comment:
We're able to delete entries only from a single service (Weblate translation memory), so the button should be conditional here.

Reply:
okay, I will add the condition.